win32ss/user/rtl/text.c

```diff
@@ -164,7 +164,7 @@ static void TEXT_Ellipsify (HDC hdc, WCHAR *str, unsigned int max_len,
     unsigned int len_ellipsis;
     unsigned int lo, mid, hi;
 #ifdef _WIN32K_
-    len_ellipsis = wcslen (ELLIPSISW);
+    len_ellipsis = (unsigned int)wcslen(ELLIPSISW);
 #else
```

Comment:
Can't we use a #define ?

Reply:
I implemented lstrlenW as an inline function instead.